packages/module/patternfly-docs/patternfly-docs.css.js

```diff
@@ -0,0 +1,8 @@
+// Patternfly
+import '@patternfly/patternfly/patternfly.css';
+// Patternfly utilities
+import '@patternfly/patternfly/patternfly-addons.css';
+// Global theme CSS
+import '@patternfly/documentation-framework/global.css';
+
+// Add your extension CSS below
```

Comment:
Don't you have to import the dist/quickstarts.css here?

Reply:
AFAIK it's not necessary. I import it in each of the markdown files where it's needed. That works to pull those styles in on the org build, and therefore the `patternfly-docs.css.js` doesn't need to be modified further in an org PR.

Reply:
Ah ok, css import in md file sounds good then